lib/auth/auth_login_test.go

```diff
@@ -815,7 +811,9 @@ func TestServer_Authenticate_headless(t *testing.T) {
 				},
 				TTL: defaults.CallbackTimeout,
 			})
-			require.NoError(t, <-errC)
+
+			// Use assert so that we also output any test failures below.
+			assert.NoError(t, <-errC)
```

Comment:
nit: Nit add a message to the assert, so we know what the errors means/where it comes from?